pymatbridge/matlab_magic.py

```diff
@@ -211,6 +211,8 @@ def matlab(self, line, cell=None, local_ns=None):
             else:
                 e_s = "There was an error running the code:\n %s"%code
                 result_dict = self.eval(code)
+        except MatlabInterperterError:
+            raise
         except:
```

Comment:
I think that @isbadawi's subsequent commit (https://github.com/arokem/python-matlab-bridge/commit/944cf22b1c121fb43b307769d0b82e8f866ed4ef) actually addresses this well. Sorry for the noise.